featuretools/__init__.py

```diff
@@ -35,11 +35,11 @@
 logger = logging.getLogger('featuretools')
 
 # Call functions registered by other libraries when featuretools is imported
-for entry_point in pkg_resources.iter_entry_points('featuretools_initialize'):
+for entry_point in pkg_resources.iter_entry_points('alteryx_open_src_initialize'):
     try:
         method = entry_point.load()
         if callable(method):
-            method()
+            method('featuretools')
```

Comment:
won't this break any existing plugins that use the `featuretools_intialize` entry point?

Reply:
- There isn't any existing plugins (that we can search for) besides the update checker that uses that entry point:
  - https://github.com/search?q=featuretools_intialize&type=code

Reply:
I could make this change be in another for loop...

Reply:
add another loop for now and float an issue on whether to remove the old entry point?

Reply:
Sounds good. Fixed the MR and issue created: https://github.com/alteryx/featuretools/issues/1461